Fix url link

Yikun opened the pull request:
I guess the link should be `http://git.openstack.org/cgit/openstack/windmill`, just a typo.

For more stable use, I change it to the github repo https://github.com/openstack/windmill

kiwik commented on config/zuul/openlab-main.yaml:
```diff
@@ -13,32 +13,22 @@
           - dtantsur/rust-openstack
           - kubernetes/cloud-provider-openstack:
               exclude-unprotected-branches: true
-          - theopenlab/ansible:
```
should be `ansible/ansible`, and the following project should be changed.

kiwik commented on config/zuul/openlab-main.yaml:
```diff
@@ -13,32 +13,22 @@
-              exclude-unprotected-branches: true
           - huaweicloud/huaweicloud-service-broker:
               exclude-unprotected-branches: true
-          - openservicebrokerapi/osb-checker
```
why delete this project?

h00130372 replied:
made a mistake when paste/copy

kiwik commented on config/zuul/openlab-main.yaml:
```diff
@@ -13,32 +13,22 @@
+              exclude-unprotected-branches: true
+          - terraform-providers/terraform-provider-openstack:
+              exclude-unprotected-branches: true
           - flystack/misty
```
I prefer to format all of project in this schema:
```
          - gophercloud/gophercloud:
              exclude-unprotected-branches: true/false
```

Yikun committed:
Fix url link

I guess the link should be `http://git.openstack.org/cgit/openstack/windmill`, just a typo.

For more stable use, I change it to the github repo https://github.com/openstack/windmill

kiwik reviewed (approved):
LGTM
